agimus_demo_05_pick_and_place/agimus_demo_05_pick_and_place/orchestrator.py

```diff
@@ -291,20 +347,20 @@ def pick_and_place(
         self.open_gripper()
         self.open_gripper()
 
-        self.publish(grasp_path)
+        self.publish(grasp_path, use_visual_servoing=True, object_name=object_name)
         rclpy.spin_until_future_complete(
             self.trajectory_publisher, self.trajectory_publisher.future_trajectory_done
         )
         if placing_path is not None:
             # TODO: check automatically
             self.close_gripper()
-            self.publish(placing_path)
+            self.publish(placing_path, use_visual_servoing=False)
```

Comment:
We can do a first trial with this but this is not the ideal case. We want to have a visual servoing during the preplacement phase.